package.json

```diff
@@ -16,7 +17,22 @@
     "copyfiles": "2.1.1"
   },
   "scripts": {
-    "build": "node ./src/convert.js",
-    "postbuild": "copyfiles -f ./src/static/* ./dist/"
+    "prettier": "prettier --write 'dist/**/*.{js,jsx,ts,tsx,json,css,scss,md}'",
```

Comment:
Yeah dunno about using Prettier for distribution files and why? Prettier is used mainly for source that is been edited.

Reply:
Fixed to touch only dynamically generated js, d.ts and scss files before rollup build. This way there is no need to care about formatting when generating the files.